src/AutoRest.CSharp/AutoRest/Communication/StandaloneGeneratorRunner.cs

```diff
@@ -90,7 +90,8 @@ private static Configuration LoadConfiguration(string basePath, string json)
                 root.GetProperty(nameof(Configuration.PublicClients)).GetBoolean(),
                 root.GetProperty(nameof(Configuration.ModelNamespace)).GetBoolean(),
                 root.GetProperty(nameof(Configuration.HeadAsBoolean)).GetBoolean(),
-                root.GetProperty(nameof(Configuration.SkipCSProjPackageReference)).GetBoolean()
+                root.GetProperty(nameof(Configuration.SkipCSProjPackageReference)).GetBoolean(),
+                root.GetProperty(nameof(Configuration.OperationGroupMapping))
```

Comment:
Can we add this to one of the readme examples / tests so that we see how it is expected to be applied